.vex/v0.41.0.vex.json

```diff
@@ -0,0 +1,22 @@
+{
+  "@context": "https://openvex.dev/ns/v0.2.0",
+  "@id": "https://inspektor-gadget.io/vex/v0.41.0.vex.json",
```

Comment:
Since we're not publishing this on the website, should it point to the release asset?